packages/core/src/app/checkout/getCheckoutStepStatuses.ts

```diff
@@ -90,16 +90,10 @@ const getBillingStepStatus = createSelector(
             : EMPTY_ARRAY;
     },
     (checkout, billingAddress, billingAddressFields) => {
-        const hasAddress = billingAddress
+        const isComplete = billingAddress
```

Comment:
Have we checked what happens for stripe in this case?

Reply:
@animesh1987 attached the screen recording with stripe payment process using GP:

https://github.com/bigcommerce/checkout-js/assets/99336044/6b10ea49-c788-45af-beeb-9eb8a8e336a3

Reply:
Thank you for posting PayPal e2e video in another comment.
Because we are not sure the impact o other payment wallet buttons, does it make sense to limit the change to PayPal only?

Reply:
As far as i understood this is the general change for all payment wallet buttons. But i think i need to make some clarification about it. I will be back with information.

Reply:
@bc-peng i have made changes related to PayPal only